ladybug_rhino/fromgeometry.py

```diff
@@ -141,8 +141,17 @@ def from_polyface3d(polyface):
 
 def from_face3d_to_wireframe(face):
     """Rhino PolyLineCurve from ladybug Face3D."""
-    return rg.PolylineCurve(
-        [from_point3d(pt) for pt in face.boundary] + [from_point3d(face.boundary[0])])
+    if face.has_holes == True:
+        hole_curves = [rg.PolylineCurve(
+            [from_point3d(pt) for pt in tup] + [from_point3d(tup[0])]) for tup in face.holes]
+        boundary_curve = [rg.PolylineCurve(
+            [from_point3d(pt) for pt in face.boundary] + [from_point3d(face.boundary[0])])]
+        curves = boundary_curve + hole_curves
+        return curves
+    else:
+        boundary_curve = [rg.PolylineCurve(
+            [from_point3d(pt) for pt in face.boundary] + [from_point3d(face.boundary[0])])]
+        return boundary_curve
```

Comment:
I think we need to be consistent now and always return a list. So can you change this to be a list with one item and also change the docstring of this method accordingly.

Reply:
@chriswmackey, I beleive, it is a list with one item. That is the curve for the boundary.